src/test/java/com/star_trello/darkside/repo/QueueRepoTest.java

```diff
@@ -0,0 +1,82 @@
+package com.star_trello.darkside.repo;
+
+import com.star_trello.darkside.model.Queue;
+import com.star_trello.darkside.model.User;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.annotation.Rollback;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+class QueueRepoTest {
+    private static final String TITLE1 = "FRONT";
+    private static final String DESCRIPTION1 = "Front tasks queue";
+    private static final String TITLE2 = "BACK";
+    private static final String DESCRIPTION2 = "Back tasks queue";
+    private static final String MAIL = "1van@mail.ru";
+    private static final String USERNAME = "1van";
+    private static final String PASSWORD = "123";
```

Comment:
Lets go extract this 3 constants in common interface where we will hold our constants for tests. Because this values already used in previous test.